tirocks-sys/src/lib.rs

```diff
@@ -150,6 +169,13 @@ impl rocksdb_Status {
     }
 }
 
+impl PartialEq for rocksdb_Status {
+    #[inline]
+    fn eq(&self, other: &Self) -> bool {
+        self.code_ == other.code_
```

Comment:
add `subcode_` and `sev_`

Reply:
The native implementation only checks for code.  https://github.com/tikv/rocksdb/blob/6.4.tikv/include/rocksdb/status.h#L371